exercise10.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
```

Comment:
The functions in this file are named relatively well, but even more so, are super short and succinct. That's what you're striving for.